test/unit/foundation.test.ts

```diff
@@ -3,20 +3,35 @@ import { expect, fixture, html } from '@open-wc/testing';
 import {
   ComplexAction,
   EditorAction,
+  identity,
   ifImplemented,
   invert,
   isCreate,
   isDelete,
   isMove,
+  isSame,
   isSimple,
   isUpdate,
   newActionEvent,
   newPendingStateEvent,
   newWizardEvent,
+  singletonTags,
+  specialTags,
 } from '../../src/foundation.js';
-
+import { getDocument } from '../data.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/openscd/open-scd/pullRequest?prid=7195001)
```suggestion
import { getDocument } from "../data.js";
```